ROADMAP.md

```diff
@@ -9,22 +9,17 @@ The version of the spec that rkt implements can be seen in the output of `rkt ve
 
 rkt's version 1.0 release marks the command line user interface and on-disk data structures as stable and reliable for external development. The (optional) API for pod inspection is not yet completely stabilized, but is quite usable.
 
-
-### rkt 1.9.0 (June)
-
-- full integration with Kubernetes (aka "rktnetes")
-- app-level seccomp support [#1614](https://github.com/coreos/rkt/issues/1614)
-- enhanced DNS configuration [#2044](https://github.com/coreos/rkt/issues/2044)
-- IPv6 support [appc/cni#31](https://github.com/appc/cni/issues/31)
-
 ### rkt 1.10.0 (July)
 
+- app-level seccomp support [#1614](https://github.com/coreos/rkt/issues/1614)
 - stable gRPC [API](https://github.com/coreos/rkt/tree/master/api/v1alpha)
 - further improvements for SELinux environments, especially Fedora in enforcing mode
```

Comment:
This is unfortunately stalled now due to https://github.com/coreos/rkt/issues/1727#issuecomment-218753806, please drop it for now.